### PR TITLE
ParseWordDoc - read Urls

### PR DESCRIPTION
**Pull request by adi88d**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-17648)

## Description
The script is now extracting urls from hyperlinks in the document.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/ReleaseNotes/1_10_17.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### ParseWordDoc
+- The script is now extracting urls from the document.
```
Comment (DeanArbel) [inline Packs/CommonScripts/ReleaseNotes/1_10_17.md:4]:
```suggestion
- The script is now extracting URLs from the document.
```

File: Packs/CommonScripts/Scripts/ParseWordDoc/ParseWordDoc_test.py [hunk 2]
```diff
@@ -56,3 +56,20 @@ def test_parse_word_doc(self, mocker):
         main()
         result = self.get_demisto_results()
         assert result.get('File') == 'file-sample.txt'
+
+
+def test_extract_urls_xml():
```
Comment (DeanArbel) [inline Packs/CommonScripts/Scripts/ParseWordDoc/ParseWordDoc_test.py:61]: Please add a test where no URLs are available.
Please also test extract_urls_docx.

**Review by DeanArbel — CHANGES_REQUESTED** [review]
Looks good, but please expand the tests.

File: Packs/CommonScripts/Scripts/ParseWordDoc/ParseWordDoc_test.py (hunk 2 above)
Comment (adi88d) [reply Packs/CommonScripts/Scripts/ParseWordDoc/ParseWordDoc_test.py]: done

**Pull request merged**